mate-panel/panel-run-dialog.c

```diff
@@ -1962,6 +2063,19 @@ panel_run_dialog_new (GdkScreen  *screen,
 	g_signal_connect_swapped (dialog->run_dialog, "destroy",
 				  G_CALLBACK (panel_run_dialog_destroy), dialog);
 
+	GtkAccelGroup* accel_group = gtk_accel_group_new();
```

Comment:
looking at my previous review, shouldn't this be unreffed after adding it to the window? I'd thing there is a leak here

Reply:
Fixed (fixup 5dbd153e35a034b233494b6ba7e7377b8d6af537).

However, `valgrind --leak-check=full ./mate-panel --run-dialog` still shows lots of leaks, but I guess this is normal?

Reply:
@mattarroz some will be normal (there's a lot of one-shot "leaks" of persistent data in e.g. freetype, some in GTK and others, but they are not really "leaks", just annoying in the reports -- you might be lucky using GTK and GLib's suppression files), but some might still be bugs.  Yet, so long as your changes don't introduce new issues you're fine for this PR.